tests/OpenToolkit.Tests/Assertions.fs

```diff
@@ -51,9 +48,13 @@ type internal Assert =
 
     static member ApproximatelyEquivalent(a : float32,b : float32) =
         if not <| approxEq a b then raise <| new Xunit.Sdk.EqualException(a,b)
+        
+    static member ApproximatelyEquivalent(a : Color4<'T>, b : Color4<'T>, epsilon:float32) =
+        if not <| approxEqEpsilon a.X b.X epsilon && approxEqEpsilon a.Y b.Y epsilon && approxEqEpsilon a.Z b.Z epsilon && approxEqEpsilon a.W b.W epsilon then
+            raise <| new Xunit.Sdk.EqualException(a,b)
 
-    static member ApproximatelyEquivalent(a : Color4, b : Color4, epsilon:float32) =
-        if not <| approxEqEpsilon a.R b.R epsilon && approxEqEpsilon a.G b.G epsilon && approxEqEpsilon a.B b.B epsilon && approxEqEpsilon a.A b.A epsilon then
+    static member ApproximatelyEquivalent(a : Color3<'T>, b : Color3<'T>, epsilon:float32) =
+        if not <| approxEqEpsilon a.X b.X epsilon && approxEqEpsilon a.Y b.Y epsilon && approxEqEpsilon a.Y b.Y epsilon then
             raise <| new Xunit.Sdk.EqualException(a,b)
 
     static member ApproximatelyEqualEpsilon(a : Vector2, b : Vector2, epsilon:float32) =
```

Comment:
Im very picky here again. Probably a `Format Document` will fix already.

Please insert space infront and after `:` at `epsilon:float32`

Reply:
Isnt this an exact duplicate comment?

Reply:
Yes, on purpose. In a review you mark single things, not 1 comment per file mentioning all the things to be changed in a file. Just saying.

Reply:
No, its literally an exact duplicate, the same line number and the same change

Reply:
That's odd. Not on purpose then, but I remember i saw that twice, hence commented twice. Maybe I was blind when i was pressing the + button to add the comment? Anyway, you got the point, I assume.